libraries/AP_Camera/AP_Camera_Servo.cpp

```diff
@@ -43,6 +51,30 @@ bool AP_Camera_Servo::trigger_pic()
     return true;
 }
 
+
+bool AP_Camera_Servo::set_zoom(ZoomType zoom_type, float zoom_value)
+{
+    if (zoom_type == ZoomType::RATE) {
```

Comment:
Hi @Williangalvani,

Thanks for this.  It seems like the absolute zoom output will be slightly incrementing or decrementing each time the set-zoom is called.  This is different (I think) to how other cameras implement the set-zoom command.  The others will continue to zoom until a follow-up set-zoom command is received

Reply:
right!
[this](https://mavlink.io/en/messages/common.html#FOCUS_TYPE_STEP) Is what I had in mind. I just updated the code and enums.

Reply:
actually I'm just fixing the rate implementation. the step one is likely to cause jitter...